build-support/bin/check_banned_imports.py

```diff
@@ -15,8 +15,10 @@ def main() -> None:
   )
   rust_files = find_files("src/rust/engine", extension=".rs")
 
+  python2_files = filter_files(python_files, snippet_regex=r"^from __future__ import")
+
   check_banned_import(
-    python_files,
+    python2_files,
```

Comment:
Maybe change to `python_2_compatible_files` to signify that they are Python2 compatible, but can also be run as python3?